docs/recipes/build/add-build-script.md

```diff
@@ -1,88 +1,256 @@
+
+        let redirect (index, (name, createProcess)) =
+            createProcess
+            |> CreateProcess.redirectOutputIfNotRedirected
+            |> CreateProcess.withOutputEvents (onStdout index name) (onStderr name)
+
+        let printStarting indexed =
+            for (index, (name, c: CreateProcess<_>)) in indexed do
+                let color = colors.[index % colors.Length]
+                let wd =
+                    c.WorkingDirectory
+                    |> Option.defaultValue ""
+                let exe = c.Command.Executable
+                let args = c.Command.Arguments.ToStartInfo
+                print color $"{name}: {wd}> {exe} {args}" ""
+
+        let run cs =
+            cs
+            |> Seq.toArray
+            |> Array.indexed
+            |> fun x -> printStarting x; x
+            |> Array.map redirect
+            |> Array.Parallel.map Proc.run
+
+let createProcess exe arg dir =
+    CreateProcess.fromRawCommandLine exe arg
+    |> CreateProcess.withWorkingDirectory dir
     |> CreateProcess.ensureExitCode
+
+let dotnet = createProcess "dotnet"
+let npm =
+    let npmPath =
+        match ProcessUtils.tryFindFileOnPath "npm" with
+        | Some path -> path
+        | None ->
+            "npm was not found in path. Please install it and make sure it's available from your path. " +
+            "See https://safe-stack.github.io/docs/quickstart/#install-pre-requisites for more info"
+            |> failwith
+
+    createProcess npmPath
+
+let run proc arg dir =
+    proc arg dir
     |> Proc.run
     |> ignore
 
-let dotnet cmd workingDir = DotNet.exec (DotNet.Options.withWorkingDirectory workingDir) cmd ""
+let runParallel processes =
+    processes
+    |> Proc.Parallel.run
+    |> ignore
 
-Target.create "Bundle" (fun _ ->
-    Shell.cleanDir deployDir
-    npm "install" clientPath
-    dotnet (sprintf "publish -c Release -o \"%s\"" deployDir) serverPath
-    npm "run build" clientPath
-)
+let runOrDefault args =
+    try
+        match args with
+        | [| target |] -> Target.runOrDefault target
+        | _ -> Target.runOrDefault "Run"
+        0
+    with e ->
+        printfn "%A" e
+        1
+```
+
+#### 4. Move the files
+
+You can now cut the `Build.fsproj`, `Build.fs` and `Helpers.fs` files and paste them in the root of your solution, then delete the Build folder as it is no longer needed.
+
+> This is to allow us to execute `dotnet run` at the root of the solution rather than navigating into a `Build` subfolder.
 
-"Clean"
-    ==> "InstallClient"
-    ==> "Bundle"
+#### 5. Add the project to the solution
 
-Target.runOrDefaultWithArguments "Bundle"
+Run the following command
+
+```bash
+dotnet sln add Build.fsproj
 ```
 
-#### 4. Build the Application
-You can now build your application by running the following command at the root directory of your solution:
-```powershell
-dotnet fake build
-```
\ No newline at end of file
+#### 6. Convert to Paket
```

Comment:
I would get rid of 6-8 and just say "add these dependencies to the project" with links to Paket and Nuget guides for adding packages.